stable/hazelcast/values.yaml

```diff
@@ -48,6 +48,19 @@ hazelcast:
   # configurationFiles are any additional Hazelcast configuration files
   # configurationFiles:
 
+# affinity specifies the affinity/anti-affinity of different pods. The commented out
+# example below shows how you could ensure your hazelcast pods are scheduled on
+# different Kubernetes nodes
+affinity:
+#  podAntiAffinity:
```

Comment:
Thanks for adding this example. The issue with this example is that it applies both to Hazelcast and Management Center. Meaning that if you have 3 nodes, then you'd end up with the following POD placement.

```
$ kubectl get pods -o wide                                                                                                                                                          NAME                               READY   STATUS    RESTARTS   AGE     IP          NODE                                                NOMINATED NODE   READINESS GATES
my-release-hazelcast-0             1/1     Running   0          8m44s   10.16.2.5   gke-rafal-test-cluster-default-pool-b413eb80-jm0f   <none>           <none>
my-release-hazelcast-1             1/1     Running   0          8m5s    10.16.1.7   gke-rafal-test-cluster-default-pool-b413eb80-fhh1   <none>           <none>
my-release-hazelcast-2             0/1     Pending   0          7m26s   <none>      <none>                                              <none>           <none>
my-release-hazelcast-mancenter-0   1/1     Running   0          8m44s   10.16.0.5   gke-rafal-test-cluster-default-pool-b413eb80-4g7h   <none>           <none>
```
Which is not correct, because there are 3 nodes and we want each Hazelcast member wants to on a separate node. Could you please change the example to be make the (anti)affinity based on both `app.kubernetes.io/name` and `role`?

Reply:
Hi @leszko - I've updated the example. Thank you for catching that detail!